src/platform/packages/shared/shared-ux/feedback_snippet/src/feedback_panel.tsx

```diff
@@ -24,6 +24,7 @@ import {
 } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import type { FeedbackView } from './feedback_snippet';
+import Confetti from './confetti/confetti';
```

Comment:
nit: prefer a barrel import.  If you're importing the non-lazy component, export it from the barrel as `ConfettiComponent`.